### PR TITLE
drop old condition table and add new one with split cdw id

### PR DESCRIPTION
**Pull request by eclendenning-va**
`[CDWIdNumber] bigint null,
  [CDWIdResourceCode] char(1) null,`
  
  These fields had to remain non null to load the current records, because `CDWIdNumber` was able to be null it couldnt be the primary key so `CDWId` is still the primary key

### CODE REVIEW

File: db/migration/V16_1__add_condition.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Create table [App].[Condition]
```
Comment (ilaflamme-va) [inline db/migration/V16_1__add_condition.sql:1]: I think this file name should be a little more descriptive even thought we haven't done a great job in the past. Perhaps
`v16_1__add_condition_split_cdw_id.sql`

**Comment by nmoisan** [comment]
LGTM, the primary key on `CDWId` should be just fine.

**Comment by jtaylor-va** [comment]
The `CDWId` as the primary key should be fine for now. There is a story for the data team to eventually remove that column completely, but I don't see that being worked anytime soon.

File: db/migration/V16_1__add_condition.sql (hunk 1 above)
Comment (jtaylor-va) [reply db/migration/V16_1__add_condition.sql:1]: I'm curious why the `DROP` and `CREATE` need to be in separate files?

**Review by jhulbert-va — APPROVED** [review]

File: db/migration/V16_1__add_condition.sql (hunk 1 above)
Comment (eclendenning-va) [reply db/migration/V16_1__add_condition.sql]: They didn't need to be but for clarity of what the update was doing I split it into a drop and create.

**Pull request merged**